qa/rpc-tests/test_framework/util.py

```diff
@@ -385,6 +385,25 @@ def start_node(i, dirname, extra_args=None, rpchost=None, timewait=None, binary=
 
     return proxy
 
+def assert_start_raises_init_error(i, dirname, extra_args=None, expected_msg=None):
```

Comment:
In the the original [PR](https://github.com/bitcoin/bitcoin/pull/9832), I see that this method is used in `wallet-hd.py` but the backport is missing this change. Looking at the current code, it looks like the call to `assert_start_raises_init_error` is actually already there, but commented out and replaced with a custom logic for the times we didn't have assert_start_raises_init_error. I assume you can now remove the custom logic and use this new method instead?